.dotnet/src/Custom/Assistants/Internal/GeneratorStubs.Internal.cs

```diff
@@ -40,6 +42,7 @@ internal partial class InternalMessageDeltaContentImageFileObject { }
 internal partial class InternalMessageDeltaContentImageUrlObject { }
 
 [CodeGenModel("MessageDeltaObjectDelta")]
+[Experimental("OPENAI001")]
```

Comment:
Q: Why is the attribute applied to some APIs here but not others? Technically we only need them on public types.

Reply:
Some of these APIs have public properties, base types, and other members that are marked as experimental. To resolve the compile-time errors, I updated all the custom code accordingly.